week01/BestTimeToBuyAndSellStock.java

```diff
@@ -0,0 +1,19 @@
+class Solution {
+
+    // TC = O(n) : where n is the size of Prices array.
+    // MC = O(1)
+
+    int inf = 1_000_000_000;
```

Comment:
Bad variable naming.
Should be static and capital.

Reply:
Fixed it.